Editor/Resources/EditorWindow/common.uss

```diff
@@ -142,4 +148,136 @@ Image.icon--large {
 .page__subtitle {
     font-size: var(--font-size-extralarge);
     margin-bottom: var(--spacing-large);
+}
+
+.foldout {
+    color: var(--color-foldout-title);
+}
+
+.foldout--hidden {
+    display: none;
+}
+
+.foldout .unity-foldout__text {
```

Comment:
Is `unity-foldout` an inbuilt class type? Why is this necessary?

Reply:
yes it is an inbuilt type. this allows us to override their default styles